### ISSUE TITLE
[Bug]: new Shallow diffusion bug?

### ISSUE THREAD

**Issue by iiallgaii** [svc-develop-team/so-vits-svc#347]
### 系统平台版本号

win7

### GPU 型号

3060

### Python版本

3.8.9

### PyTorch版本

1.13.1

### sovits分支

4.0(默认)

### 数据集来源（用于判断数据集质量）

UVR

### 出现问题的环节或执行的命令

train Shallow diffusion

### 情况描述

May i know that is it same content between train.txt and val.txt ? the below  files train.txt & val.txt are auto gen by Sovits, why i have 10 wav but only 2 wav show in val.txt? is it mean Sovits will only train  
./dataset/44k/Ai_mary/ocalsV2_8.wav and ./dataset/44k/Ai_mary/ocalsV2_10.wav ? thanks !

train.txt

./dataset/44k/Ai_mary/ocalsV2_1.wav
./dataset/44k/Ai_mary/ocalsV2_2.wav
./dataset/44k/Ai_mary/ocalsV2_3.wav
./dataset/44k/Ai_mary/ocalsV2_4.wav
./dataset/44k/Ai_mary/ocalsV2_5.wav
./dataset/44k/Ai_mary/ocalsV2_6.wav
./dataset/44k/Ai_mary/ocalsV2_7.wav
./dataset/44k/Ai_mary/ocalsV2_8.wav
./dataset/44k/Ai_mary/ocalsV2_9.wav
./dataset/44k/Ai_mary/ocalsV2_10.wav

val.txt

./dataset/44k/Ai_mary/ocalsV2_8.wav
./dataset/44k/Ai_mary/ocalsV2_10.wav

### 日志

```python
-
```

**Comment by bfloat16** [comment]
On the contrary, the audio in train.txt is the training set, while the audio in val.txt is the validation set. 
According to your description, those 10 audio clips will be used for training.

**Comment by iiallgaii** [comment]
> On the contrary, the audio in train.txt is the training set, while the audio in val.txt is the validation set. According to your description, those 10 audio clips will be used for training.

i see , thanks!

**Comment by HuanLinOTO** [comment]
close as completed
